tests/contrib/pylibmc/test.py

```diff
@@ -386,3 +391,39 @@ def test_patch_unpatch(self):
         spans = self.pop_spans()
         assert spans, spans
         assert len(spans) == 1
+
+
+class TestPylibmcPatchServiceNames(TestPylibmcPatchDefault):
+    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_SERVICE="mysvc", DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v0"))
```

Comment:
Kind of unrelated? But I guess that might just be because I view these as separate to some degree. It all has to get in, so should be fine I think.

Reply:
I split up the operation and service name changes originally, but they are all under the same umbrella of "service name" changes.  I'm going to leave this in for now